google-cloud-bigquery/test/google/cloud/bigquery/copy_job_test.rb

```diff
@@ -96,7 +96,8 @@ def copy_job_hash id
         "tableId"   => "target_table_id"
       },
       "createDisposition" => "CREATE_IF_NEEDED",
-      "writeDisposition" => "WRITE_EMPTY"
+      "writeDisposition" => "WRITE_EMPTY",
+      "operationYype" => nil
```

Comment:
Typo?
(Also, what are we testing here, if this was still passing even with the typo?)

Reply:
We are not testing anything on the operation_type here hence the tests are passing. Removed as operation type is an optional field on job config.